crates/moon/src/cli/test.rs

```diff
@@ -78,6 +78,9 @@ pub struct TestSubcommand {
 
     #[clap(flatten)]
     pub auto_sync_flags: AutoSyncFlags,
+
+    #[clap(long)]
+    pub no_parallelize: bool,
```

Comment:
just use `no-parallel`

Maybe we need `--jobs` flag?

Reply:
will make at next PR